src/utils/hashes/index.ts

```diff
@@ -44,32 +50,56 @@ const currencyToHex = (currency: string): string => {
     return bytesToHex(bytes);
   }
   return currency;
-};
+}
 
-const addLengthPrefix = (hex: string): string => {
+function addLengthPrefix(hex: string): string {
   const length = hex.length / 2;
   if (length <= 192) {
     return bytesToHex([length]) + hex;
   }
   if (length <= 12480) {
     const x = length - 193;
+    // eslint-disable-next-line no-bitwise -- adding a prefix to hex requires bitwise operations
     return bytesToHex([193 + (x >>> 8), x & 0xff]) + hex;
   }
   if (length <= 918744) {
     const x = length - 12481;
+    // eslint-disable-next-line no-bitwise -- adding a prefix to hex requires bitwise operations
```

Comment:
you can probably just disable this `no-bitwise` at the top of the file, since it is widely used throughout the file